packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -219,6 +222,7 @@ export class NotifiFrontendClient {
     private _configuration: NotifiFrontendConfiguration,
     private _service: NotifiService,
     private _storage: NotifiStorage,
+    private _wsUrl: string
```

Comment:
I would not put the wsUrl here since it is the constant. 
Now all the endpoint related operations are in `notifi-graphql`'s `GraphQLClient`. Do you think would it be a good idea to move it here?